db/db.go

```diff
@@ -145,30 +145,114 @@ func Init(username, password, protocol, address, database string) error {
 
 func setupDb(tx *sql.Tx) error {
 	queries := []string{
-		`CREATE TABLE IF NOT EXISTS accounts (uuid BINARY(16) NOT NULL PRIMARY KEY, username VARCHAR(16) UNIQUE NOT NULL, hash BINARY(32) NOT NULL, salt BINARY(16) NOT NULL, registered TIMESTAMP NOT NULL, lastLoggedIn TIMESTAMP DEFAULT NULL, lastActivity TIMESTAMP DEFAULT NULL, banned TINYINT(1) NOT NULL DEFAULT 0, trainerId SMALLINT(5) UNSIGNED DEFAULT 0, secretId SMALLINT(5) UNSIGNED DEFAULT 0)`,
+		`CREATE TABLE IF NOT EXISTS accounts (
```

Comment:
you need to write a new query, which add those column, cause

> IF NOT EXISTS

> Prevents an error from occurring if the table exists. However, there is no verification that the existing table has a structure identical to that indicated by the [CREATE TABLE](https://dev.mysql.com/doc/refman/8.0/en/create-table.html) statement.

Reply:
This also does not belong in the accounts table.

Reply:
No tables are altered, changes for existing are only golang syntax to make it more readable (using multilines to not have very long lines)
The new query is on line 247